libraries/apollo-api/api/apollo-api.api

```diff
@@ -305,7 +306,7 @@ public final class com/apollographql/apollo3/api/CompiledField : com/apollograph
 	public final fun getType ()Lcom/apollographql/apollo3/api/CompiledType;
 	public final fun nameWithArguments (Lcom/apollographql/apollo3/api/Executable$Variables;)Ljava/lang/String;
 	public final fun newBuilder ()Lcom/apollographql/apollo3/api/CompiledField$Builder;
-	public final fun resolveArgument (Ljava/lang/String;Lcom/apollographql/apollo3/api/Executable$Variables;)Ljava/lang/Object;
+	public final fun resolveArgument (Ljava/lang/String;Lcom/apollographql/apollo3/api/Executable$Variables;)Lcom/apollographql/apollo3/api/Optional;
```

Comment:
Same here. An argument with a variable value is possibly absent if that variable is absent as well and unless we change the signature, there's no way to account for that. We could go through deprecation cycles but given this is a low level API, I'm hoping this is fine.